pkg/ktime/ktime_windows.go

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package ktime
+
+import (
+	"syscall"
+	"time"
+	"unsafe"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+var (
+	dll            = syscall.MustLoadDLL("kernel32.dll")
+	queryCounter   = dll.MustFindProc("QueryPerformanceCounter")
+	queryFrequency = dll.MustFindProc("QueryPerformanceFrequency")
+)
+
+func ToProto(ktime uint64) *timestamppb.Timestamp {
```

Comment:
Functions `ToProto`, `ToProtoOpt`, and `DiffKtime` seem to be the same across both versions. Any reason for not adding them to `pkt/ktime/ktime.go`?